pkg/helper/client.go

```diff
@@ -5,17 +5,26 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"os"
 	"os/exec"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/gobwas/glob"
```

Comment:
Why don't we use golang regex ? 
https://gobyexample.com/regular-expressions

Reply:
IMO regular expressions are overkill in this case. They are hard to read and write. Do you see any case where regex would be needed? All I see now is filtering out stuff with name prefix.

Reply:
For this case we don't need complicated regex veth.* so there is no need for another third party that user has to learn.

veth.* vs veth* this is not that different

Reply:
Also what we are filtering here is an arbitrary name not url or path, so better go with regex than globing.

Reply:
Globing is usually used to filter stuff like github.com/kubevirt/* or the same for some path, while regex is mor suitable for names for example, a user want to filter a range of interfaces for whatever reason you cannot do it with globing, but you can do it with regex eth[0-5] or the like.

Reply:
Bah let's go with gobling, is good enough, if we need more complex scenarios we can add a fix

Reply:
You had a fair point though. I don't have a strong opinion on this. @HlinaCZ won't be around next weeks, so we have a time to think this through.